data-prepper-plugins/saas-source-plugins/confluence-source/src/main/java/org/opensearch/dataprepper/plugins/source/confluence/utils/ConfluenceConfigHelper.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+package org.opensearch.dataprepper.plugins.source.confluence.utils;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.opensearch.dataprepper.plugins.source.confluence.ConfluenceSourceConfig;
+import org.opensearch.dataprepper.plugins.source.source_crawler.utils.AddressValidation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.opensearch.dataprepper.plugins.source.confluence.utils.Constants.BASIC;
+import static org.opensearch.dataprepper.plugins.source.confluence.utils.Constants.OAUTH2;
+
+/**
+ * The type Confluence configuration.
+ */
+@Slf4j
+public class ConfluenceConfigHelper {
```

Comment:
I think all of these things can be moved into the configuration classes themselves to have consistent access into them.